recipes/pygmo/bld.bat

```diff
@@ -0,0 +1,22 @@
+mkdir build
+cd build
+
+cmake ^
+    -G "%CMAKE_GENERATOR%" ^
+    -DCMAKE_PREFIX_PATH=%LIBRARY_PREFIX% ^
+    -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+    -DPAGMO_WITH_EIGEN3=yes ^
+    -DPAGMO_WITH_NLOPT=yes ^
+    -DPAGMO_INSTALL_HEADERS=no ^
+    -DPAGMO_BUILD_PYGMO=yes ^
+    ..
+
+cmake --build . --config Release
+
+cmake --build . --config Release --target install
+
+start /b ipcluster start
+
+timeout 20
+
+python -c "import pygmo; pygmo.test.run_test_suite()"
```

Comment:
I put the ``ipcluster stop`` command in the unix CI bits because all builds there are run in the same VM, and so the builds after the first one would fail because the script would attempt to start another ``ipcluster`` while the ``ipcluster`` daemon from the previous build was still running.

I'll add the termination command here as well for symmetry.